packages/server/src/tests/header/headerSelection.ts

```diff
@@ -28,41 +27,32 @@ const nonSupportersTestNonUK: HeaderTest = {
 };
 
 const nonSupportersTestUS = (): HeaderTest => {
-    const givingTuesdayStart = new Date('2021-11-29T17:00:00'); //remove "Subscribe" Monday 12:00 PM EST
-    const givingTuesdayEnd = new Date('2021-12-01T09:00:00'); //re-add "Subscribe" on Wednesday morning GMT
-    const currentDateTime = new Date();
-    const shouldShowSubscribeButton = !(
-        currentDateTime >= givingTuesdayStart && currentDateTime <= givingTuesdayEnd
-    );
+    const nyeStart = new Date('2021-12-31T00:00:00');
+    const nyeEnd = new Date('2022-01-02T00:00:00');
```

Comment:
These times are in UTC. I thought it might just be simplest to remove it for the whole of 31st & 1st UTC to make sure it's hidden in the US for the whole of the 31st.